include/votca/xtp/atomcontainer.h

```diff
@@ -70,9 +70,12 @@ class AtomContainer {
   }
 
   void AddContainer(const AtomContainer<T>& container) {
+    Index offset = _atomlist.size();
     _type += "_" + container._type;
-    _atomlist.insert(_atomlist.end(), container._atomlist.begin(),
-                     container._atomlist.end());
+    for (auto at : container._atomlist) {
```

Comment:
```suggestion
    for (const auto& at : container._atomlist) {
```

Reply:
actually I am not sure about other `T` like `atom` or `staticsite`. I am not sure if this does not break other template arguements.